### PR TITLE
Feature/css disabled

### PR DESCRIPTION
**Pull request by jcarres-mdsol**
Do not follow links that have been disabled with CSS
@mdsol/team-10

### CODE REVIEW

File: lib/grell/page.rb [hunk 1]
```diff
@@ -205,17 +205,9 @@ def has_selector?(selector)
...
-        enabled_links = anchors_in_body.reject { |anchor| anchor.disabled? }
-
-        unique_links = enabled_links.map do |anchor|
-         anchor['href'] || anchor['data-href']
-        end.compact
-
-        unique_links.map{|link| link_to_url(link)}.uniq.compact
+        links =  @rawpage.all_anchors.map { |anchor| Link.new(anchor) }
+        body_enabled_links = links.reject { |link| link.inside_header? || link.disabled? }
+        body_enabled_links.map{|link| link.to_url(host)}.uniq.compact
```
Comment (ssteeg-mdsol) [inline lib/grell/page.rb]: Spacing between the map { } is needed

File: lib/grell/page.rb [hunk 2]
```diff
@@ -224,37 +216,56 @@ def all_links
...
+            else
+              href # Absolute link to our own host
+            end
+          else
+            if uri.path.nil?
+              Grell.logger.debug "GRELL does not follow links without a path: #{uri}"
+              nil
+            end
+            if uri.path.start_with?('/')
+              host + href  #convert to full URL
```
Comment (ssteeg-mdsol) [inline lib/grell/page.rb]: I know it's copy paste code, but there's a bunch of comments missing a space between the # and the text

**Comment by jcarres-mdsol** [comment]
@ssteeg-mdsol  addressed

**Comment by ykitamura-mdsol** [comment]
looks good to me

**Pull request merged**